traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -178,71 +178,71 @@
           },
           "ccrDnsTtl": {
             "title": "Delivery Service DNS TTL",
-            "desc": "The Time To Live on the DNS record for the Traffic Router A and AAAA records."
+            "desc": "(For DNS routed delivery services only) The Time To Live on the DNS record for the Traffic Router A and AAAA records. Default value is 30 seconds. Setting too high or too low will result in poor caching performance."
           },
           "cdnId": {
             "title": "CDN",
-            "desc": "The CDN in which the delivery service belongs to."
+            "desc": "The CDN in which the delivery service belongs."
           },
           "checkPath": {
             "title": "Check Path",
             "desc": "A path (ex: /crossdomain.xml) to verify the connection to the origin server with. This can be used by Check Extension scripts to do periodic health checks against the delivery service."
           },
           "deepCachingType": {
             "title": "Deep Caching",
-            "desc": "When to do deep caching for this delivery service.<br><br>See <a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#rl-deep-cache\" target=\"_blank\">Deep Caching</a> for more details..."
+            "desc": "Enables clients to be routed to the closest possible deep edge caches on a per Delivery Service basis.<br><br><a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#rl-deep-cache\" target=\"_blank\">See Deep Caching</a>"
           },
           "displayName": {
             "title": "Display Name",
-            "desc": "A human-readable name for the delivery service."
+            "desc": "Name of the service that appears in the Traffic portal. No character restrictions."
           },
           "dnsBypassCname": {
             "title": "DNS Bypass CNAME",
-            "desc": "Description required"
+            "desc": "(For DNS routed delivery services only) Domain name to overflow requests when the the max Bps or Max Tps for this delivery service are exceeded."
           },
           "dnsBypassIp": {
             "title": "DNS Bypass IP",
-            "desc": "This is the address to respond to A requests with when the max Bps or Max Tps for this delivery service are exceeded."
+            "desc": "(For DNS routed delivery services only) IPv4 address to overflow requests when the the max Bps or Max Tps for this delivery service are exceeded."
```

Comment:
again, ditch the (For DNS routed delivery services only)

and is it max or Max?